albumentations/augmentations/geometric/resize.py

```diff
@@ -67,20 +67,23 @@ def __init__(self, max_size=1024, interpolation=cv2.INTER_LINEAR, always_apply=F
         self.interpolation = interpolation
         self.max_size = max_size
 
-    def apply(self, img, interpolation=cv2.INTER_LINEAR, **params):
-        return F.longest_max_size(img, max_size=self.max_size, interpolation=interpolation)
+    def apply(self, img, max_size=1024, interpolation=cv2.INTER_LINEAR, **params):
```

Comment:
Could you please add typing to all function parameters?

I know that it was not there before, but we are slowly moving toward having them everywhere.

Reply:
Added typing

Reply:
Hi @ternaus, can you take another look? Thanks